EMMO/README.md

```diff
@@ -0,0 +1,74 @@
+# EMMO
+This [W3ID](https://w3id.org) provides a persistent URI namespace for the [Elementary Multiperspective Material Ontology (EMMO)](https://github.com/emmo-repo/EMMO/) and its domain ontologies.
+
+## Redirection Rules
+This section contains a general summary of the logic behind the redirection rules.
+
+1. `https://w3id.org/emmo`
```

Comment:
This system is case-sensitive.  Either change these to `EMMO` or move the files to an `emmo` dir.